src/main/java/com/griefcraft/listeners/LWCPlayerListener.java

```diff
@@ -759,8 +759,8 @@ public void onInventoryClick(InventoryClickEvent event) {
             return;
         }
 
-        // If it's not a donation or display chest, ignore it
-        if (protection.getType() != Protection.Type.DONATION && protection.getType() != Protection.Type.DISPLAY) {
+        // If it's not a donation, supply or display chest, ignore it
```

Comment:
for consistency (display is before supply here in code)
```suggestion
        // If it's not a donation, display, or supply chest, ignore it
```

Reply:
Applied suggestion locally to reduce number of commit, thanks.